data-prepper-core/src/test/java/org/opensearch/dataprepper/parser/model/PipelineConfigurationTests.java

```diff
@@ -67,78 +84,16 @@ public void testPipelineConfigurationCreation() {
-
-        assertTrue(exception.getMessage().contains(expected));
-    }
-
-    private void assertEqualProcessorPluginSettings(
-            final List<Map.Entry<String, Map<String, Object>>> expectedPluginSettings,
-            final List<PluginSetting> actualPluginSettings) {
-        assertEquals(expectedPluginSettings.size(), actualPluginSettings.size());
-
-        expectedPluginSettings.forEach(expectedSetting -> {
-            final PluginSetting actualSetting = actualPluginSettings.stream()
-                    .filter(plugin -> expectedSetting.getKey().equals(plugin.getName()))
-                    .findFirst()
-                    .orElseThrow(() -> new InputMismatchException("Expected setting named " + expectedSetting.getKey()));
-
-            final Map<String, Object> expectedSettingValue = expectedSetting.getValue();
-            final Set<String> expectedKeySet = expectedSettingValue.keySet();
-            final Map<String, Object> settings = actualSetting.getSettings();
-            assertEquals(expectedKeySet.size(), settings.size());
-
-            expectedKeySet.forEach(key -> {
-                assertEquals(expectedSettingValue.get(key), settings.get(key));
-            });
-        });
-    }
-
-    @Test
-    public void testPipelineConfigurationWithPrepperOrProcessorAreEquivalent() {
-        Map.Entry<String, Map<String, Object>> sourcePluginSettings = TestDataProvider.validSingleConfiguration();
-        List<Map.Entry<String, Map<String, Object>>> expectedPluginSettings = TestDataProvider.validMultipleConfigurationOfSizeOne();
-
-        PipelineConfiguration prepperConfig = new PipelineConfiguration(
-                sourcePluginSettings,
-                null,
-                expectedPluginSettings,
-                null,
-                TestDataProvider.validMultipleConfiguration(),
-                TestDataProvider.TEST_WORKERS,
-                TestDataProvider.TEST_DELAY);
-        PipelineConfiguration processorConfig = new PipelineConfiguration(
-                sourcePluginSettings,
-                null,
-                null,
-                expectedPluginSettings,
-                TestDataProvider.validMultipleConfiguration(),
-                TestDataProvider.TEST_WORKERS,
-                TestDataProvider.TEST_DELAY);
-
-        assertEqualProcessorPluginSettings(expectedPluginSettings, prepperConfig.getProcessorPluginSettings());
-        assertEqualProcessorPluginSettings(expectedPluginSettings, processorConfig.getProcessorPluginSettings());
-    }
-
-    @Test
-    public void testOnlySourceAndSink() {
-        final PipelineConfiguration pipelineConfiguration = new PipelineConfiguration(TestDataProvider.validSingleConfiguration(),
-                null,
-                null,
-                TestDataProvider.validMultipleConfigurationOfSizeOne(),
-                null, null);
+    void testOnlySourceAndSink() {
+        sinks = TestDataProvider.validMultipleConfigurationOfSizeOne();
```

Comment:
Why are we not continuing to use `validMultipleConfiguration` for `sinks`?

Reply:
I don't think it matters. But, the original test was using size of one. So, I kept it that way. I wanted to keep the tests as similar as possible to their original intent.